db/migrations/202407151352_autoincrement.go

```diff
@@ -0,0 +1,80 @@
+package migrations
+
+import (
+	_ "embed"
+
+	"github.com/go-gormigrate/gormigrate/v2"
+	"gorm.io/gorm"
+)
+
+// This migration (inside a DB transaction),
+// - Adds AUTOINCREMENT to the primary key of:
+// - apps, app_permissions, request_events, response_events, user_configs
+//
+// request_events and response_events are not critical (and also payments are dropped in the same release)
+// so we just drop those tables and re-create them.
+var _202407151352_autoincrement = &gormigrate.Migration{
+	ID: "202407151352_autoincrement",
+	Migrate: func(db *gorm.DB) error {
+
+		if err := db.Transaction(func(tx *gorm.DB) error {
+
+			// drop old request and response event tables
+			if err := tx.Exec(`
+DROP TABLE request_events;
+DROP TABLE response_events;
+`).Error; err != nil {
+				return err
+			}
+
+			// User configs - create new table, copy old values, delete old table, rename new table
+			if err := tx.Exec(`
+CREATE TABLE "user_configs_2" ("id" integer PRIMARY KEY AUTOINCREMENT, "key" text NOT NULL UNIQUE, "value" text, "encrypted" numeric, created_at datetime,updated_at datetime);
+INSERT INTO user_configs_2 (id, key, value, encrypted, created_at, updated_at) SELECT id, key, value, encrypted, created_at, updated_at FROM user_configs;
+DROP TABLE user_configs;
+ALTER TABLE user_configs_2 RENAME TO user_configs;
```

Comment:
wondering if we should do this in a second migration? those are the most valuable data. 
and those actually also don't have associations thus the auto increment is not that important.

Reply:
@bumi yeah, I am wondering if I should just remove this from the migration and we just leave this table without autoincrement?

Reply:
Decision: let's not update this one